### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by pastorp3**
## Calculator Milestone 5: Event handlers
- Added event handler for buttons
- Implemented logic with event handler
- Added reactive display

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars, class-methods-use-this,  react/destructuring-assignment */
```
Comment (SmithVyne) [inline src/components/App.js]: You shouldn't be disabling these Lints. They are required to improve your code quality.

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  handleClick(btnName) {
+    const { total, next, operator } = this.state;
+    this.setState(state => calculate(state, btnName));
+  }
+
+  render() {
+    return (
+      <div>
+        <Display onInput={this.state.next ? this.state.next : this.state.total} />
```
Comment (SmithVyne) [inline src/components/App.js]: Use object destructuring instead of using `this` repetitively.

**Review by SmithVyne — CHANGES_REQUESTED** [review]
### Changes Required 🤕 👍 
Hello, your project is nice. You just need to make a few additional improvements to get it approved. 

[REQUIRED] - Do not disable the Linters and improve your code quality.

**Comment by pastorp3** [comment]
- Added destructuring object to render

Note: Only disable no-unused-state on App.js because It shows a conflict with the operator in the state but I can not delete the operator from the state

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @pastorp3, congratulations on fixing the comments outlined by the previous TSE :+1: Awesome :smile: The object destructuring used effectively, and also the linter disable reduced and explained :+1: I hereby approve the project

[Optional]
- Kindly improve the code commit messages going forward, kindly use `add` instead of `added`, `fix` instead of `fixed`, `import` instead of `imported`, `implement` instead of `implemented` etc
- It would be nice if we add a screenshot of the app
- Going forward, let us try to implement GitHub flow, for example; create a development branch from the master and also create the event branch from the development. So by this, we can merge to development but not master directly

Kindly go ahead and merge into development, if you really implemented the GitHub/git flow
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**